modules/text_server_adv/gdextension_build/methods.py

```diff
@@ -73,31 +73,6 @@ def disable_warnings(self):
         self.Append(CXXFLAGS=["-w"])
 
 
-def make_icu_data(target, source, env):
```

Comment:
Won't work, this copy is for the GDExtension build, which does not include global `methods`, so `Sconstruct` in the gdextension_build should have code similar to the changes in SCsub to load it. And I'm not sure it won't conflict with other GDExtension stuff.

Reply:
What would you recommend is the best way to run into this error?

Reply:
You can run GDExtension build by cloning [`godot-cpp`](https://github.com/godotengine/godot-cpp) into `modules/text_server_adv/gdextension_build` and running `scons` from `modules/text_server_adv/gdextension_build`.

Reply:
Ty @bruvzg ; I'll get it all working